### ISSUE TITLE
Set relative impact in earnings variation chart to 0% when 0/0

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-app#1508]
Currently anything divided by 0 is null, but I think 0/0 can be 0 instead ([example](https://policyengine.org/us/household?focus=householdOutput.earnings&reform=51058&region=dc&timePeriod=2024&baseline=2&household=43023))
<img width="789" alt="image" src="https://github.com/PolicyEngine/policyengine-app/assets/6076111/4cd4f3e1-b49a-4481-a18d-f4d08b908fea">

https://github.com/PolicyEngine/policyengine-app/blob/49bd6aa0c9781e4de93ff60d7dd16ce31e44d7e6/src/pages/household/output/EarningsVariation/BaselineAndReformChart.jsx#L395

**Comment by anth-volk** [comment]
To clarify, the UX issue for the user here is that, when hovering over the point at (0,0), the chart indicates that DC TANF has changed "by $0," whereas elsewhere, the chart indicates that it "hasn't changed," which is caused by the division rules you described above, correct?

**Comment by MaxGhenis** [comment]
No, points beyond x=8,000 or so have no value because their DC TANF is $0 today

**Comment by anth-volk** [comment]
Ah, I thought your chart was merely zoomed in, now I understand

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
